web/template/choose_address.gohtml

```diff
@@ -1,12 +1,28 @@
 {{ template "page" . }}
 
+
+
 {{ define "pageTitle" }}
+    {{ $manualTitle := "personsAddress" }}
```

Comment:
thoughts on passing in a `Labels` struct or something to describe the labels that can be overridden for professional certificate providers vs. `.IsProfessionalCertificateProvider`?

Reply:
I'm also noticing if I go back to the page the title isn't showing the "work" labels

Reply:
<img width="683" alt="image" src="https://github.com/ministryofjustice/opg-modernising-lpa/assets/146390/2afe455c-c896-4150-b75f-92da89377778">

and wondering if this should be listed as "Work address"?